src/test-utils.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { IntlProvider } from 'react-intl';
+
+import { initializeStore } from 'store';
+
+export const executeThunk = async (thunk, dispatch, getState) => {
+  await thunk(dispatch, getState);
+};
+
+export function renderWithProviders(
+  ui,
+  {
+    preloadedState = {},
+    // Automatically create a store instance if no store was passed in
```

Comment:
@01001110J Is this comment necessary?

Reply:
Not really, should I deleted? Could be useful